NYCFireWire/NYCFireWire/src/Views/IncidentTableViewCell.swift

```diff
@@ -8,17 +8,27 @@
 
 import UIKit
 
-class IncidentTableViewCell: UITableViewCell {
+protocol LikeButtonDelegate {
+    func incidentLiked()
+}
 
+class IncidentTableViewCell: UITableViewCell {
     
     @IBOutlet weak var boroLabel: UILabel!
     @IBOutlet weak var boxLabel: UILabel!
     
     @IBOutlet weak var titleLabel: UILabel!
     @IBOutlet weak var subtitleLabel: UILabel!
     
-    @IBOutlet weak var addressLabel: UILabel!
-    @IBOutlet weak var timeLabel: UILabel!
+    
+    @IBOutlet weak var likeButton: UIButton!
+    @IBOutlet weak var numberOfLikesLabel: UILabel!
+    
+    @IBOutlet weak var numberOfCommentsLabel: UILabel!
+    
+    var incident: Incident?
```

Comment:
Views should try its best to be stateless, so in the cellForIndexPath should be setting the current isLiked or not.
We shouldn't store incident in this view